mmdet3d/structures/bbox_3d/base_box3d.py

```diff
@@ -275,12 +275,13 @@ def in_range_3d(
             Tensor: A binary vector indicating whether each point is inside the
             reference range.
         """
-        in_range_flags = ((self.tensor[:, 0] > box_range[0])
```

Comment:
`bbox3d.in_range_3d` should filter out bboxes where the gravity center of the bbox is not in the target range, not the bottom center. Btw, this function not use the training/testing pipeline of any other model, except for DSVT. Therefore, this modification does not cause BC-breaking.